DEVELOPING.md

```diff
@@ -58,13 +58,7 @@ To run device tests there are few options:
   - Open the resulting `MapboxMaps.xcodeproj`.
   - Test (Cmd-U) the `MapboxTestHost` scheme.
 
-2. Trigger via CI by adding `[run device tests]` to a git commit message. These tests also run "nightly". 
-(Examples can also be run as tests on CI by adding `[run app device tests]`.)
-
-3. Trigger tests on AWS Device Farm from the command line by running `make test-with-device-farm SCHEME=MapboxTestHost APP_NAME=MapboxTestHost`. This requires certain environment variables to be set; please see the makefile for these.
-
-4. Trigger tests on a local device (connected by USB) using the same setup as Device Farm testing by running:
-`make local-test-with-device-farm-ipa SCHEME=MapboxTestHost CONFIGURATION=Release ENABLE_CODE_SIGNING=1`
+2. Trigger via CI by manual approve of `optional-tests` workflow. These tests also run "nightly".
 
 ## Integration Tests
 
```

Comment:
There is still a reference to the AWS Device Farm on line 65(just below this one), not sure what's the scope of this removal, but should it be replaced as well with Firebase Test Lab?